src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+    >
+      <div class="search-field search-field--big">
```

Comment:
where is tag `form`?

Reply:
Tag `form` is parent for `<div class="search-field search-field--big">`.
Could you clarify the error, please.